portfolio.js

```diff
@@ -185,6 +185,17 @@ const error = document.querySelector('#error');
 const form = document.querySelector('#contact-form');
 const fullName = document.querySelector('#full-name');
 const email = document.querySelector('#email');
+const message = document.querySelector('#message');
+
+const contactKey = 'CONTACT';
+const contactDetails = localStorage.getItem(contactKey);
+
+if (contactDetails) {
+  const contact = JSON.parse(contactDetails);
+  fullName.value = contact.fullName;
+  email.value = contact.email;
+  message.value = contact.message;
+}
```

Comment:
_**OPTIONAL**_

You might want to ensure that the localStorage checks and all is only done when the page loads. One way of implementing that is using the `window.onload` feature:
```JS
window.onload = () => {
  const contactKey = 'CONTACT';
  const contactDetails = localStorage.getItem(contactKey);

  if (contactDetails) {
    const contact = JSON.parse(contactDetails);
    fullName.value = contact.fullName;
    email.value = contact.email;
    message.value = contact.message;
  }
};
```